packages/editor/src/components/post-locked-modal/index.js

```diff
@@ -148,6 +148,10 @@ function PostLockedModal() {
 		return null;
 	}
 
+	if ( window.__experimentalEnableSync ) {
```

Comment:
I'm trying to think about what this PR would look like when we want to get this into core. Would this change be accepted by the Gutenberg folks, given instead of doing `window.<access flag here>` they are adding a filter for plugin users? If it is accepted, what would it look like when we want to move it out of the experiment stage and have it be a core feature? How would null be returned then?